src/main/java/org/osc/sdk/controller/element/Element.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.sdk.controller.element;
+
+public interface Element {
+	/**
+	 * The parentId represents the port pair id in case of SFC
```

Comment:
please update java doc. Element id should contain information on what it is not what the parentId is.

Give example of neutron port id and SFC. In case of SFC, is this the flow classifer id? or just neutron port id etc

Ideally we would like to avoid references to specific implementations, but since this is javadoc only it should be acceptable.